test/data/example.html

```diff
@@ -16,7 +16,8 @@
           <span itemprop="name">
             <a itemprop="url" href="http://www.lib.ncsu.edu">NCSU Libraries</a>
           </span>
-        </span>.
+        </span>. Born in 
+        <span itemprop="birthDate" content="1900-09-14T21:30">Sep 14</span>
```

Comment:
Also `<time itemprop="birthDate" datetime="1900-09-14T21:30">Sep 14</time>` is good, if that helps.